lib/animina/validations/delete_about_story.ex

```diff
@@ -22,14 +23,13 @@ defmodule Animina.Validations.DeleteAboutStory do
 
     user_id = Ash.Changeset.get_attribute(changeset, opts[:user])
 
-    stories = get_stories_for_user(user_id)
-
+  
     case Headline.by_id(headline_id) do
       {:ok, headline} ->
-        if length(stories) == 1 && headline.subject == "About me" do
+        if  headline.subject == "About me" do
```

Comment:
This might be controversial but I think that `case` feels a bit more Elixir like than `if`.

```elixir
      case  headline.subject do
        "About me" -> 
          {:error,
           field: opts[:attribute],
           message: "You cannot delete the 'About me' story."}
        _ -> 
          :ok
        end
```